spec/models/miq_group_spec.rb

```diff
@@ -144,64 +144,62 @@
   end
 
   context "Testing active VM aggregation" do
+    let (:miq_group) { FactoryGirl.create(:miq_group, :description => "test group") }
```

Comment:
There shouldn't be a space between `let` and `(`.  Removing it should fix most of the rubocop comments.